testing/python_lib/build_config.py

```diff
@@ -62,18 +124,53 @@ def _create_scale_faucet_config(self, t1_switches, t2_switches, access_ports):
         dps = {}
         for dp_index, dp_name in enumerate(t1_dps):
             tap_vlan = test_vlan if not dp_index else None
-            interfaces = self._build_t1_interfaces(dp_index, t1_dps, t2_dps,
-                                                   50 + dp_index * 2, [setup_vlan], tap_vlan)
+            interfaces = self._build_dp_interfaces(
+                dp_index, dps=t1_dps, t2_dps=t2_dps, tagged_vlans=[setup_vlan],
+                tap_vlan=tap_vlan, egress_port=FAUCET_EGRESS_PORT, lacp=True)
             dps[dp_name] = self._build_datapath_config(
-                177 + dp_index, ('0e:00:00:00:01:%s' % ("{:02x}".format(dp_index+1))), interfaces)
+                T1_DP_ID_START + dp_index, interfaces,
+                (T1_DP_MAC_PREFIX + "{:02x}".format(dp_index+1)))
```

Comment:
NB: I see this bit of code 3x+ times, so is a good candidate for extracting to a function.  Right now it takes a bit of mental energy to parse what this is doing, so having a named function would clarify for the reader.

Reply:
Done